packages/flutter_tools/lib/src/desktop_device.dart

```diff
@@ -0,0 +1,173 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
```

Comment:
Instead import base/io.dart to make the tests happy

Reply:
Someday I'll figure out how to stop having this done automagically for me by VS Code :P